portal2-internal/util/math/base.hpp

```diff
@@ -1,429 +1,17 @@
 #pragma once
 
-// https://gist.github.com/es3n1n/1601c7b74d03abe36ee2cc5790409ccc
 namespace util::math {
     constexpr auto e = 2.71828182845904523536f;
     constexpr auto pi = 3.14159265358979323846f;
```

Comment:
i think it would be better to use https://en.cppreference.com/w/cpp/header/numbers for constants here

something like
```suggestion
    constexpr auto pi = std::numbers::pi_v<double>;
```

same for others

Reply:
lgtm, done